server/app/models/grid_domain_authorization.rb

```diff
@@ -9,6 +9,9 @@ class GridDomainAuthorization
   field :domain, type: String
   field :challenge, type: Hash
   field :challenge_opts, type: Hash # TODO encrypt?
+  field :authorization_type, type: String
+
+  has_one :grid_service
```

Comment:
This field is never set in the `GridCertificates::AuthorizeDomain` mutation, so it's not used, and always nil?